tests/antares/services/api_services/test_study_api.py

```diff
@@ -215,6 +217,15 @@ def test_read_study_api(self):
             mocker.get(renewable_url, json=[])
             mocker.get(thermal_url, json=[])
             mocker.get(storage_url, json=[])
+            mocker.get(
+                output_url,
```

Comment:
For these 2 tests, you can just return an empty list: `mocker.get(output_url, json=[])` as you don't really test the content